server/eclipse-project/script/build-release

```diff
@@ -0,0 +1,31 @@
+# make changes like https://github.com/FreeAndFair/ColoradoRLA/pull/353
```

Comment:
Indeed!  I started pulling this out last night. Here it is:

https://github.com/FreeAndFair/ColoradoRLA/pull/356 Help build the deliverable zip: build_release etc

It might also make sense to have a template release branch with the necessary changes, e.g. from https://github.com/FreeAndFair/ColoradoRLA/pull/353/commits/41be78ffafb89667ff57e4d1a84a7289ae48310c
that can be rebased onto master when ready to branch off a release.